src/main/kotlin/no/nav/syfo/melding/database/MeldingQuery.kt

```diff
@@ -130,14 +130,19 @@ fun DatabaseInterface.updateInnkommendePublishedAt(uuid: UUID) {
     }
 }
 
-// TODO: Etterhvert må også denne utelukke de meldingene med meldingstype=påminnelse, siden vi ikke skal sende en påminnelse på en påminnelse
 const val queryGetUbesvarteMeldinger =
     """
         SELECT DISTINCT m_utgaende.*
         FROM MELDING m_utgaende
         WHERE NOT m_utgaende.innkommende
             AND m_utgaende.ubesvart_published_at IS NULL
             AND m_utgaende.created_at <= ?
+            AND m_utgaende.type != 'FORESPORSEL_PASIENT_PAMINNELSE'
+            AND m_utgaende.id NOT IN (
```

Comment:
Tror det blir bedre med en NOT EXISTS ( ..) sånn at m_utgaende.id kan brukes i where-clause'n til subselect'en.

Reply:
Aha, så noe a la dette:
```sql
AND NOT EXISTS (
                SELECT melding_id
                FROM melding_status
                WHERE melding_id = m_utgaende.id AND status = 'AVVIST'
            )
```

Reply:
Ja, det ser riktig ut 🏆